step-templates/file-system-create-file.json

```diff
@@ -1,52 +1,59 @@
 {
   "Id": "175a91a9-562e-49b9-bfa6-609a4e16bc56",
   "Name": "File System - Create File",
-  "Description": "Creates a file, using the full path that is provided.",
+  "Description": "Creates a file under specified directory and write given content.",
   "ActionType": "Octopus.Script",
-  "Version": 2,
+  "Version": 3,
+  "CommunityActionTemplateId": null,
   "Properties": {
-    "Octopus.Action.Script.ScriptBody": "$filePath = $OctopusParameters['FilePath']\n$fileContent = $OctopusParameters['FileContent']\n$encoding = $OctopusParameters['Encoding']\n\nNew-Item -ItemType file -Path $filePath -Value '' -force\n\nif(![string]::IsNullOrEmpty($fileContent))\n{\n  Set-Content -path $filePath -value $fileContent -encoding $encoding\n}",
-    "Octopus.Action.Script.Syntax": "PowerShell"
+    "Octopus.Action.Script.ScriptBody": "$fileName = $OctopusParameters['FileName']\n$path = $OctopusParameters['Path']\n$content = $OctopusParameters['Content']\n\nif ($fileName -and $path) {\n\n if(-Not(Test-Path $path))\n    {\n       Write-Error \"$path does not exist\"\n       exit 1\n    }\n\n    $fullPath=Join-Path $path -ChildPath $fileName\n    \n    Write-Host \"File Path: $fullPath\"\n    \n    New-Item -type file -Path $fullPath -force\n\n    if($content){\n       Add-Content $fullPath $content\n    }\n\n    Write-Host \"$fileName created at $path\"\n}\nelse{\n    Write-Error \"File creation failed : Incorrect File name/path.\"\n    exit 1\n}\n",
```

Comment:
Previously, this replaced the file content if the file existed. It now appends.
This could be a breaking change for some users.